department-of-veterans-affairs/vets-api
# Stale Benefit Intake Submission Logging

## Summary

refactor benefits_intake_status_job to report on stale submissions

## Related issue(s)

https://github.com/department-of-veterans-affairs/va.gov-team/issues/85842

## Testing done

- create a form_submission and related attempt
- set created_at time of attempt to a day in distant past
- run the benefits_intake_status_job
- there should be a statsd increment for the form type with 'stale' and Rails log

## What areas of the site does it impact?

any app using LH BenefitsIntake API and the FormSubmission, FormSubmissionAttempt tracking

## Acceptance criteria
